packages/lbd/PKGBUILD

```diff
@@ -3,18 +3,17 @@
 
 pkgname=lbd
 pkgver=20130719
-pkgrel=5
+pkgrel=6
 groups=('blackarch' 'blackarch-recon')
 pkgdesc='Load Balancing detector,'
 url='http://ge.mine.nu/code/lbd'
 arch=('any')
 license=('GPL')
-depends=('bash' 'gnu-netcat')
+depends=('bash' 'dnsutils' 'openbsd-netcat')
```

Comment:
Why is `dnsutils` required?

Reply:
@noraj Because lbd needs `host` command for working. `host` is provided by `dnsutils`.